dotcom-rendering/src/web/components/Caption.tsx

```diff
@@ -163,9 +172,10 @@ export const Caption = ({
 					iconStyle(palette),
 					format.display === ArticleDisplay.Immersive &&
 						hideIconBelowLeftCol,
+					isVideo && videoIconStyle,
```

Comment:
Maybe it makes more sense to have `CameraIcon` and `VideoIcon` as their own components here? It will read nicer and offers up a much nicer path for future expansion if we want to have other icons types or different colours, etc. in the future

Reply:
Sure, that would be cleaner 👍🏻